openquake/calculators/event_based_risk.py

```diff
@@ -59,8 +61,7 @@ def build_agg_curve(lr_list, insured_losses, ses_ratio, curve_resolution,
         if len(data) == 0:  # realization with no losses
             continue
         for i in range(insured_losses + 1):  # insured_losses
```

Comment:
No, it would do a needless computation of insured curves starting from NaN values if insured_losses is false.